spec/javascripts/spec/track_click.spec.js

```diff
@@ -24,14 +24,15 @@ describe('A click tracker', function() {
       spyOn(root.GOVUKAdmin, 'trackEvent');
       tracker.start(element);
       element.find("a.foo").click();
-      expect(GOVUKAdmin.trackEvent).toHaveBeenCalledWith('button-pressed', 'Foo');
+
+      expect(GOVUKAdmin.trackEvent).toHaveBeenCalledWith('button-pressed', 'Foo', undefined, undefined);
```

Comment:
The `undefined` will be removed when the method call will be changed to:

```GOVUKAdmin.trackEvent(options)```

See card [here](https://trello.com/c/BW5fA4ar/141-normalizes-ga-tracking-for-admin-gem)